collective/portlet/content/contentportlet.py

```diff
@@ -226,17 +198,17 @@ def body(self):
 
         # Currently nothing stops you from trying to get text from an Image
         if hasattr(self.content, 'getText'):
-            text = self.content.getText()
+            text = self.content.getText
```

Comment:
The brain probably does not have getText; for this you probably need the real content item